app/src/services/object.js

```diff
@@ -117,11 +117,13 @@ const service = {
    */
   searchObjects: async (params, etrx = undefined) => {
     let trx;
+    let response = [];
     try {
       trx = etrx ? etrx : await ObjectModel.startTransaction();
 
-      const response = await ObjectModel.query(trx)
-        .allowGraph('version')
+      response.data = await ObjectModel.query(trx)
+        .allowGraph('[objectPermission, version, bucketPermission]')
+        .groupBy('object.id')
```

Comment:
Optional: May be worth putting a comment about why groupBy was chosen over orderBy for future reference.